README.md

```diff
@@ -1,9 +1,64 @@
 # ASU Unity Design System (UDS)
+
+## Table of Contents
+
+- [ASU Unity Design System (UDS)](#asu-unity-design-system-uds)
+  - [Table of Contents](#table-of-contents)
+  - [Quickstart Guide](#quickstart-guide)
+  - [Visual Regression Testing](#visual-regression-testing)
```

Comment:
@davidornelas11 This PR is almost good to go except let's move the Visual Regression section to right before the "Running end-to-end testing" section. That would mean moving line 8 to right above line 25 (or 24 when line 8 is removed 😛 ) and then moving the related block of content to the same position in the body of the document.